raven/utilities/forecasting.py

```diff
@@ -191,6 +191,7 @@ def get_ECCC_dataset(climate_model):
 
     if climate_model == "GEPS":
         file_url = "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/datasets/forecasts/eccc_geps/GEPS_latest.ncml"
+        file_url = "https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/testdata/geps_forecast/GEPS_latest.ncml"
```

Comment:
Which one is good?

Reply:
It's the second one. I left the first one there because eventually that will be the correct path, but for now it's the second one. Perhaps I should have commented it out instead of just overwriting with no info...

Also, I have modified the code a bit to fit both this branch and the hindcasting branch, I'll push something probably in the next hour.

Reply:
ok sure! if you could comment the first line I think it would make the intention clearer indeed, because to my eyes, it's instantly suspicious (some languages even go as far as having that a syntax error, which I personnaly find is making sense).

Reply:
Got it! Will do. Thanks!

It's good that you keep an eye on me, I'm bound to do stupid stuff once in a while 😄